### ISSUE TITLE
Updates to 1.16

### ISSUE THREAD

**Issue by hfoxy** [pandapaul/minecraft-items#4]
Would it be possible to get this updated for 1.16? Alternatively, provide the steps used to generate the files and I will create a PR with the updates.

**Comment by pandapaul** [comment]
Yeah sure thing.  Let me make sure that the source data has also been updated, but assuming it has I'll get this rebuilt with the latest stuff soon.
